src/Http/Response.php

```diff
@@ -167,8 +167,8 @@ class Response implements ResponseInterface
         'gz' => 'application/x-gzip',
         'bz2' => 'application/x-bzip',
         '7z' => 'application/x-7z-compressed',
+        'hal' => ['application/hal+xml', 'application/vnd.hal+xml'],
         'haljson' => ['application/hal+json', 'application/vnd.hal+json'],
-        'halxml' => ['application/hal+xml', 'application/vnd.hal+xml'],
```

Comment:
Wasn't this added in 4.2? Renaming it will break any applications using it. We could add another alias to align with what apache has though.

Reply:
Yes. Wasn't sure if fixing the name now and documenting would be ok.

Reply:
But I'll just add the alias